conf/iptables-rules.v4

```diff
@@ -0,0 +1,14 @@
+*filter
+:INPUT ACCEPT [100:6731]
+:FORWARD ACCEPT [0:0]
+:OUTPUT ACCEPT [125:46180]
+:LOG_DROP - [0:0]
+-A INPUT -i lo -j ACCEPT
+-A INPUT -s 104.236.86.218/32 -p tcp -m tcp --dport 27017 -m state --state NEW,ESTABLISHED -j ACCEPT
+-A INPUT -s 127.0.0.1/32 -p tcp -m tcp --dport 27017 -m state --state NEW,ESTABLISHED -j ACCEPT
+-A INPUT -s 162.243.11.137/32 -p tcp -m tcp --dport 27017 -m state --state NEW,ESTABLISHED -j ACCEPT
+-A INPUT -s 104.238.128.74/32 -p tcp -m tcp --dport 27017 -m state --state NEW,ESTABLISHED -j ACCEPT
```

Comment:
There are 3 IPs listed here. What are they? I thought there was only one app server here. A few problems I see with this:

* If these are all app servers, this config file becomes a barrier to scale because new machines can't connect.
* ...which makes me think maybe we can take advantage of ansible, because ansible would let us loop over the IPs of machines in a particular role
* ...but then that makes me think "Can we just set up a rule for the subnet, so we don't even have to do that?"